docs/Using-Fleet/fleetd.md

```diff
@@ -312,6 +255,76 @@ fleetctl package
 ```
 If this setting is not used, you will need to configure client TLS certificates on devices' browsers.
 
+#### Debug
+
+You can use the `--debug` option in `fleetctl package` to generate installers in "debug mode." This mode increases the verbosity of logging for orbit and osqueryd (log DEBUG level).
+
+## Orbit
+
+Orbit is an [osquery](https://github.com/osquery/osquery) runtime and autoupdater that is bundled in Fleetd. With Orbit, it's easy to deploy osquery, manage configurations, and stay up to date. Orbit eases the deployment of osquery connected with a [Fleet server](https://github.com/fleetdm/fleet) and is a (near) drop-in replacement for osquery in a variety of deployment scenarios.
```

Comment:
@eashaw, looking at this again, I'm not sure this is right. I think this paragraph is referring to "Fleetd" and should be moved back into the page intro (with "Orbit" changed to "Fleetd.") If I understand correctly, Fleetd is the runtime and auto-updater, and Orbit is the "Fleet-compatible version of osquery" as described in the article https://fleetdm.com/announcements/introducing-orbit-your-fleet-agent-manager.

Is that correct, @zhumo?

Reply:
Hey Mike, that article was written back when Orbit was the name of what we now call Fleetd. But the problem is that Orbit encompasses osquery and fleet desktop and another agent (which we now call orbit). So Fleetd is just the name of the bundle. Orbit is one of the agents in that bundle that does all of runtime and auto-updater. There is no fleet-compatible version of osquery. Vanilla osquery can be used with fleet. We may need to change the naming of that article or at least put a big note at the top that says we've renamed orbit to fleetd.

Reply:
@zhumo, thanks for clearing that up. Yes, we do need to update the article. I was looking at it to help me find a good intro paragraph to add to this doc to promote the value of Fleetd (which is currently missing here, as discussed yesterday).